### ISSUE TITLE
Security Vulnerability in  hapijs / hoek

### ISSUE THREAD

**Issue by jameswilson** [BluesparkLabs/gulp-frontend-tasks#3]
> We found a potential security vulnerability in one of your dependencies.
> A dependency defined in package-lock.json has known security vulnerabilities and should be updated.

Package hapijs / hoek has a known security vulnerability in version 2.16.3

**Comment by jameswilson** [comment]
The dependency graph for hoek is the following:

```
$ npm ls hoek
gulp-frontend-tasks@0.0.1 /Users/jameswilson/App/gulp-frontend-tasks
└─┬ gulp-sass@3.2.1
  └─┬ node-sass@4.9.0
    └─┬ request@2.79.0
      └─┬ hawk@3.1.3
        ├─┬ boom@2.10.1
        │ └── hoek@2.16.3  deduped
        ├── hoek@2.16.3 
        └─┬ sntp@1.0.9
          └── hoek@2.16.3  deduped
```

The node-sass package at version 4.9.0 is causing the problem and our dependency on gulp-sass cannot be updated until node-sass fixes their dependency, which is still outstanding (see link to node-sass issue queue on link below).

https://github.com/dlmanning/gulp-sass/issues/687

**Comment by jameswilson** [comment]
I believe this was fixed by #10
